core/src/main/java/org/ehcache/core/spi/cache/StoreOpStatus.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.core.spi.cache;
+
+/**
+ * @author Abhilash
+ *
+ */
+public interface StoreOpStatus {
+
+  /**
+   * Put operation status
+   */
+  public enum PutStatus implements StoreOpStatus {
```

Comment:
I would move all these enums inside the `Store` interface and drop their common super type which is not used anywhere.